app/model/DigitalEdition.scala

```diff
@@ -0,0 +1,25 @@
+package model
+
+import configuration.Config
+
+case class DigitalEdition(id: String, name: String, price: String, cmp: String)
```

Comment:
Actually, what is `cmp`? I know you just moved this code rather than writing it but am wondering if we could improve the name of this field so that it's more self-explanatory